core/templates/dev/head/css/oppia.css

```diff
@@ -574,6 +574,17 @@ textarea {
 .oppia-navbar-dropdown > li > a {
   color: #009688;
 }
+
+/* smooth color chang to the navbar-tabs */
```

Comment:
`Ease color changes for navbar tabs.`

Reply:
So are you saying for the navbar tabs i should reduce the animation time?

Reply:
Nope, this is just a comment wording change so it's clearer what this is doing.

Reply:
ok